### PR TITLE
CCMSPUI-425 | Replace all buttons with govuk dialect

### PR DESCRIPTION
**Pull request by arunkumar461**

### CODE REVIEW

**Review by farrell-m — APPROVED** [review]
Looks good, just need to update the snapshot version.

File: src/main/resources/messages.properties [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+site.continue=Continue
+site.saveAndContinue=Save and continue
+site.confirm=Confirm
+site.next=Next
+site.search=Search
+site.cancel=Cancel
+site.upload=Upload
+
+proceedings.priorAuthority=Add prior authority
+application.register=Register new client
```
Comment (PhilDigitalJustice) [inline src/main/resources/messages.properties:15]:
```suggestion
client.register=Register new client
```

File: src/main/resources/templates/application/application-client-search-no-results.html [hunk 2]
```diff
@@ -18,9 +19,7 @@ <h1 class="govuk-heading-l">
             new client' button.
         </p>
         <form action="#" th:action="@{/application/client/results}" method="post">
-            <button class="govuk-button" data-module="govuk-button">
-                Register new client
-            </button>
+            <govuk:button type="submit" th:text="#{application.register}"/>
```
Comment (PhilDigitalJustice) [inline src/main/resources/templates/application/application-client-search-no-results.html:22]:
```suggestion
            <govuk:button type="submit" th:text="#{client.register}"/>
```

File: src/main/resources/templates/application/application-client-search-results.html [hunk 3]
```diff
@@ -51,9 +52,7 @@ <h2 class="govuk-heading-m">
             </h2>
             <p class="govuk-body">If your search has not returned the client you were looking for, you can register a new
                 client by clicking the 'Register new client' button.</p>
-            <button type="submit" class="govuk-button" data-module="govuk-button">
-                Register new client
-            </button>
+            <govuk:button type="submit" th:text="#{application.register}"/>
```
Comment (PhilDigitalJustice) [inline src/main/resources/templates/application/application-client-search-results.html:55]:
```suggestion
            <govuk:button type="submit" th:text="#{client.register}"/>
```

**Review by PhilDigitalJustice — CHANGES_REQUESTED** [review]

**Review by PhilDigitalJustice — APPROVED** [review]

**Pull request merged**